e2e/config/config.go

```diff
@@ -376,6 +382,9 @@ func (c configTests) configGlobal(t *testing.T) {
 			e2e.WithProfile(tt.profile),
 			e2e.WithDir(tt.cwd),
 			e2e.PreRun(func(t *testing.T) {
+				if tt.addRequirementsFn != nil {
+					tt.addRequirementsFn(t)
+				}
 				setDirective(t, tt.directive, tt.directiveValue)
```

Comment:
@cclerget - is this okay, or is it better in an additional e2e.PreRun? Presumably I'd need a `nil` check still somehow there?

Reply:
Yes it's ok